rawkit/raw.py

```diff
@@ -7,6 +7,7 @@
 import random
 import string
 import tempfile
+import numpy as np
```

Comment:
Yah, I just meant a normal Python array. It won't be very efficient, but I've also opened an issue (#106) for discussing the best way to handle numpy support, which is something we want to do.